packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts

```diff
@@ -80,17 +79,35 @@ export async function getSmartContract(connection: IFabricGatewayConnection, sma
                             channel: thisChannelName,
                             label: chaincode.name + '@' + chaincode.version,
                             transactions: contract.transactions,
+                            contractName: contract.name,
```

Comment:
```
contractName: contract.name,
namespace: contract.name
```

Not sure we need this.

Reply:
So it's a clearer in the code when we're dealing with multiple contracts, they shouldn't have anything to do with the namespace. imo just happens that namespace also uses the `contract.name`.